app/frontend/src/test/fixtures/users.json

```diff
@@ -9,7 +9,7 @@
     "name": "Funny Dog",
     "userId": 2,
     "username": "funnydog",
-    "avatarUrl": "funnydog.jpg"
+    "avatarUrl": ""
```

Comment:
Had to change this for the basic smoke test for the group chat view to see that an initial is being displayed when no avatar URL is available.